packages/astro-embed-baseline-status/icons/browsers/chrome.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="21" height="21">
+	<path fill="#fff" d="M10.2 15.3a5.1 5.1 0 1 0 0-10.2 5.1 5.1 0 0 0 0 10.2Z"/>
+	<path fill="#229342" d="M5.783 12.75 1.367 5.103A10.198 10.198 0 0 0 10.201 20.4l4.416-7.65a5.098 5.098 0 0 1-8.834.001Z"/>
+	<path fill="#FBC116" d="M14.617 12.75 10.2 20.4a10.199 10.199 0 0 0 8.831-15.3H10.2a5.1 5.1 0 0 1 4.418 7.65Z"/>
+	<path fill="#229342" d="M10.2 14.238a4.038 4.038 0 1 0 0-8.075 4.038 4.038 0 0 0 0 8.075Z"/>
+	<path fill="#E33B2E" d="M10.2 5.1h8.833a10.198 10.198 0 0 0-17.666.002l4.416 7.649h.002a5.1 5.1 0 0 1 4.415-7.65Z"/>
+</svg>
```

Comment:
This icon looks wrong. See [baseline-status#12](https://redirect.github.com/web-platform-dx/baseline-status/issues/12).

<img width="113" alt="" src="https://github.com/user-attachments/assets/59320f3d-2698-4b3e-878b-b0409fffff2c">


Here's the correct one:

```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="-10 -10 276 276">
	<linearGradient id="a" x1="145" x2="34" y1="253" y2="61" gradientUnits="userSpaceOnUse">
		<stop offset="0" stop-color="#1e8e3e"/>
		<stop offset="1" stop-color="#34a853"/>
	</linearGradient>
	<linearGradient id="b" x1="111" x2="222" y1="254" y2="62" gradientUnits="userSpaceOnUse">
		<stop offset="0" stop-color="#fcc934"/>
		<stop offset="1" stop-color="#fbbc04"/>
	</linearGradient>
	<linearGradient id="c" x1="17" x2="239" y1="80" y2="80" gradientUnits="userSpaceOnUse">
		<stop offset="0" stop-color="#d93025"/>
		<stop offset="1" stop-color="#ea4335"/>
	</linearGradient>
	<circle cx="128" cy="128" r="64" fill="#fff"/>
	<path fill="url(#a)" d="M96 183.4A63.7 63.7 0 0 1 72.6 160L17.2 64A128 128 0 0 0 128 256l55.4-96A64 64 0 0 1 96 183.4Z"/>
	<path fill="url(#b)" d="M192 128a63.7 63.7 0 0 1-8.6 32L128 256A128 128 0 0 0 238.9 64h-111a64 64 0 0 1 64 64Z"/>
	<circle cx="128" cy="128" r="52" fill="#1a73e8"/>
	<path fill="url(#c)" d="M96 72.6a63.7 63.7 0 0 1 32-8.6h110.8a128 128 0 0 0-221.7 0l55.5 96A64 64 0 0 1 96 72.6Z"/>
</svg>
```

Reply:
Thanks! Fixed ✅